### DISCUSSION TITLE
After update Alerts don't clear

### DISCUSSION THREAD

**Discussion by SanibelJack** [Security-Onion-Solutions/securityonion#2390] [Unsupported Versions]
I could really use some help here... After update to 2.3.10 I see the Alerts on the alerts page but when I try to acknowledge them I get a 404 page not found, only filebeats is running on forward node, I can't get the other services to start.  Any help would be appreciated.

Jack

**Comment by SanibelJack** [comment]
Additional info.... From the SO webpage from the Master it does not see my sensor...

**Comment by SanibelJack** [comment]
I could really use some help here...  All containers on the Forward node except for "Docker" and "so-filebeat" are in an error state.

**Reply by weslambert** [reply]
Have you tried checking the logs in /opt/so/log for more clues?  Have you also tried a reboot?

**Reply by weslambert** [reply]
I see there is a duplicate thread here: https://github.com/Security-Onion-Solutions/securityonion/discussions/2395.

**Comment by SanibelJack** [comment]
Hi Wes,
The MySQL  says I’m out of drive space. Suggestions on how to clear some space?

Jack Voth
CISSP, CISM, CEH
Sanibel Cyber Technology, LLC
Jack.voth@sanibelcybertech.com<mailto:Jack.voth@sanibelcybertech.com>
www.sanibelcybertechnology.com<http://www.sanibelcybertechnology.com>
239-249-9311
